tools/mason/MasonPublish.chpl

```diff
@@ -481,6 +496,17 @@ proc check(username : string, path : string, trueIfLocal : bool, ci : bool) thro
     writeln(spacer);
   }
 
+  if package {
+    writeln('Checking git tag version formatting:');
+    if gitTagVersionCheck(projectCheckHome) {
+      writeln('   Valid git tag version formatting, can be published to a registry. (PASSED)');
+    } else {
+      writeln('   Invalid git tag version formatting. (FAILED)');
```

Comment:
Is it possible for this error message to include the tag(s) found to help users debug their issue?